src/config/model.rs

```diff
@@ -110,59 +110,6 @@ impl Account {
         }
     }
 
-    /// This is a little helper-function like which uses the the name and email
-    /// of the account to create a valid address for the header of the envelope
-    /// of an msg.
-    ///
-    /// # Example 1: With name
-    /// Suppose the name field in the account struct *has* a value:
-    ///
-    /// ```rust
-    /// use himalaya::config::model::Account;
-    ///
-    /// fn main() {
-    ///     let account = Account {
-    ///         // we just need those two values
-    ///         name: Some(String::from("Name")),
-    ///         email: String::from("BestEmail@Ever.lol"),
-    ///         ..Account::default()
-    ///     };
-    ///
-    ///     // get the address of the account
-    ///     let address = account.get_full_address();
-    ///
-    ///     assert_eq!("Name <BestEmail@Ever.lol>".to_string(), address);
-    /// }
-    /// ```
-    ///
-    /// # Example 2: Without name
-    /// Suppose the name field in the account-struct *hasn't* a value:
-    ///
-    /// ```rust
-    /// use himalaya::config::model::Account;
-    ///
-    /// fn main() {
-    ///     let account = Account {
-    ///         // we just need those two values
-    ///         name: None,
-    ///         email: String::from("BestEmail@Ever.lol"),
-    ///         ..Account::default()
-    ///     };
-    ///
-    ///     // get the address of the account
-    ///     let address = account.get_full_address();
-    ///
-    ///     assert_eq!("BestEmail@Ever.lol".to_string(), address);
-    /// }
-    /// ```
-    pub fn get_full_address(&self) -> String {
-        if let Some(name) = &self.name {
-            format!("{} <{}>", name, self.email)
-        } else {
-            format!("{}", self.email)
```

Comment:
This case should never happen, because the name is a mandatory option at the config level. That's why it's not present in `Config::address`.

Reply:
Aye